clients/cobol-lsp-vscode-extension/server/package-linux/note.md

```diff
@@ -0,0 +1,26 @@
+<!---
+ * Copyright (c) 2022 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its  * subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License * 2.0
+ * which is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+-->
+
+# COBOL LSP server running information
+Put in this folder the COBOL LSP linux native build for server
+
+```
+server
+```
+In Linux environment be sure that the server folder have read/write permission, in case simply run the command
```

Comment:
Should we change the read/write permission of server directory? using command? Just a thought!